tests/testthat/test-test_r_scripts.R

```diff
@@ -4,3 +4,25 @@ test_that("stop_on_error stops on error", {
   expect_error(stop_on_error(exit_code = 1), "error")
   expect_error(stop_on_error(exit_code = 99), "error")
 })
+
+test_that("example data is available under sample_data/", {
+  # FIXME: @jacobvjk, can you explain the role of this file?
+  # At https://bit.ly/35Xypsw I read:
+  # > copy TestPortfolio_Input.csv from sample_files/20_input_files/ to
+  # > working_dir/20_Raw_Inputs/
+  # Do we copy that file manually or programmatically? If so, where is the code?
+  # My experiments suggest web_tool_script_1.R runs to the end without it, and
+  # I need to know it that's okay of if I should throw an error ASAP.
+  example_data <- here::here(
+    "sample_files", "20_input_files", "TestPortfolio_Input.csv"
+  )
+  expect_true(fs::file_exists(example_data))
+})
```

Comment:
@jacobvjk, this PR is still WIP but this section is ready for your feedback.

Reply:
manually.... `TestPortfolio_Input.csv` is an example of what a user's portfolio input file would look like... they are expected to put that file into the`20_Raw_Inputs/` directory under their "working directory" (which could be named different than "working_dir")

I would expect `web_tool_script_1.R` to work as expected without `sample_files/20_input_files/TestPortfolio_Input.csv`

I would expect `web_tool_script_1.R` to NOT work as expected without `working_dir/20_Raw_Inputs/TestPortfolio_Input.csv` OR another csv file in the same directory with a filename that matches a YML file's filename in `10_Parameter_File` and has `portfolio_name_in` inside the YML set to something that matches, e.g. `working_dir/10_Parameter_File/TestPortfolio_Input_PortfolioParameters.yml` contains `portfolio_name_in: TestPortfolio_Input`